packages/p2p/src/components/modal-manager/modals/buy-sell-modal/buy-sell-modal.jsx

```diff
@@ -22,7 +22,7 @@ import { useStores } from 'Stores';
 import BuySellForm from 'Components/buy-sell/buy-sell-form.jsx';
 import BuySellFormReceiveAmount from 'Components/buy-sell/buy-sell-form-receive-amount.jsx';
 import NicknameForm from 'Components/nickname-form';
-import AddPaymentMethodForm from 'Components/my-profile/payment-methods/add-payment-method/add-payment-method-form.jsx';
+import AddPaymentMethodForm from 'Components/add-payment-method-form';
 import { api_error_codes } from 'Constants/api-error-codes';
 import { useModalManagerContext } from 'Components/modal-manager/modal-manager-context';
```

Comment:
re-arrange the orders 👀

Reply:
Ah i didn't rearrange these cause this will be refactored later 👍